iotedgehubdev/errors.py

```diff
@@ -45,7 +45,19 @@ def __init__(self, status_code, value):
         self.status_code = status_code
 
     def message(self):
-        return ("Code:{0}. Detail:{1}").format(self.status_code, self.value)
+        return ('Code:{0}. Detail:{1}').format(self.status_code, self.value)
 
     def status(self):
         return self.status_code
+
+
+class RegistriesLoginError(Exception):
+    def __init__(self, registries, errmsg):
+        self.registries = registries
+        self.errmsg = errmsg
+
+    def message(self):
+        return ('Fail to login {0}. Detail: {1}').format(self.registries, self.errmsg)
+
+    def getRegistries(self):
```

Comment:
`getRegistries` -> `get_registries`, or maybe this whole method is unnecessary because `self.registries` can be directly accessed.

Reply:
prefer to have a method to wrap against the member. Consider if we would like to return extra info or hide some info. method is a better interface to hide the details